scripts/ci/attach_ebs_cache.sh

```diff
@@ -153,6 +153,8 @@ fi
 mkdir -p /var/lib/docker
 mount $BLKDEVICE /var/lib/docker
 service docker restart
+# clear our images cache
+rm -rf /var/lib/docker/images
```

Comment:
Isn't this running every time we mount the cache on an instance? Is it ok to clear the cache then?